extensions/amp-base-carousel/1.0/base-carousel.js

```diff
@@ -71,6 +73,11 @@ export function BaseCarousel(props) {
         disabled={disableForDir(1)}
         advance={() => advance(1)}
       />
+      {pagination && (
+        <Pagination current={curSlide} goTo={setRestingIndex} {...pagination}>
```

Comment:
Overall, it feels like it'd be better if we just passed the `<Pagination  ... />` as a prop, instead of just props (subject to our offline discussion, of course). Among other reasons why: this would also make the binary footprint smaller to anyone who doesn't need `Pagination`. Otherwise, with the `pagination &&` it has to be always baked in.

Reply:
👍